package.json

```diff
@@ -32,8 +32,8 @@
     "server": "pipenv run python network-api/manage.py runserver",
     "start": "run-p build-uncompressed server watch:**",
     "test:procfile": "node test/test-procfile.js",
-    "test:eslint:js": "eslint --config ./.eslintrc.json \"cypress/integration/*.js\" \"network-api/networkapi/{,!(frontend)/**/}*.js\" \"source/js/**/*.js\" \"source/js/**/*.jsx\" webpack.config.js",
-    "test:eslint:a11y": "eslint --config ./.eslintrc.a11y.json \"source/js/**/*.jsx\"",
+    "test:eslint:js": "eslint --no-error-on-unmatched-pattern --config ./.eslintrc.json \"cypress/integration/*.js\" \"network-api/networkapi/{,!(frontend)/**/}*.js\" \"source/js/**/*.js\" \"source/js/**/*.jsx\" webpack.config.js",
```

Comment:
Becasue `\"network-api/networkapi/{,!(frontend)/**/}*.js\"` now doesn't match anything, `eslint` will raise error unless we pass `--no-error-on-unmatched-pattern` in. 

We can either remove `\"network-api/networkapi/{,!(frontend)/**/}*.js\"` or add that new option in.

Reply:
fixed!